.github/workflows/on_build_do_test.yml

```diff
@@ -56,6 +56,10 @@ jobs:
             targetUrl: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
 
       - uses: actions/checkout@v4
+        with:
+          sparse-checkout: ".github"
+          sparse-checkout-cone-mode: false
```

Comment:
Don't we want the default of `true` here? From the git docs:

```quote
The subset of files is chosen by providing a list of directories in cone mode (the default), or by providing a list of patterns in non-cone mode.
```

Reply:
true, thanks